### PR TITLE
Sanic: capture responseHeadersOnEntrySpans

### PR DESCRIPTION
**Pull request by GSVarsha**
Sanic: capture responseHeadersOnEntrySpans

Signed-off-by: Varsha GS <varsha.gs@ibm.com>

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -348,4 +348,4 @@ workflows:
             - python312
             - py39cassandra
             - py39couchbase
-            - py39gevent
+            - py39gevent
```
Comment (Ferenc-) [inline .circleci/config.yml:351]: Why the change here? In general we prefer newlines at the end of the file.

Comment (GSVarsha) [reply .circleci/config.yml]: I think I removed it by mistake. Fixed this in the latest commit.

**Review by pvital — APPROVED** [review]
It looks good to me.

**Review by Ferenc- — APPROVED** [review]
Approved
